doc/md/concepts/url.mdx

```diff
@@ -88,6 +89,20 @@ Connecting to a local PostgreSQL with credentials and SSL disabled:
 postgres://postgres:pass@0.0.0.0:5432/database?search_path=public&sslmode=disable
 ```
 
+</TabItem>
+<TabItem value="sqlserver">
+
+Connecting to a default schema of current user:
+```shell
+sqlserver://sa:P@ssw0rd0995@localhost:1433?database=master&mode=schema
+```
+
+Connecting to a local SQLServer database named `master` (all schemas). The user need to have `db_owner` role:
+```shell
+sqlserver://sa:P@ssw0rd0995@localhost:1433?database=master&mode=database
```

Comment:
```suggestion
sqlserver://sa:pass@localhost:1433?database=master&mode=database
```
Just for simplicity and keep consistent with other examples.

Reply:
```
This password needs to include at least 8 characters of at least three of these four categories: uppercase letters, lowercase letters, numbers, and non-alphanumeric symbols.
```